.gitignore

```diff
@@ -64,7 +54,7 @@ scripts/travisci_deploy_key
 fastlane/play-private-key.json
```

Comment:
and this line could also move down to `# android keystores`, and we could probably remove the `travisci_deploy_key` line, too.